docs/content/community/getting-help.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Getting Help | Dagster
+description: Have questions about how to use Dagster? Hit a bug? Have a feature request? This page includes tips on what to do.
+---
+
+# Guide to Getting Help
+
+Have questions about how to use Dagster? Hit a bug? Have a feature request? The Dagster Labs team and the Dagster community make a best-effort attempt to respond, on a few different platforms.
+
+For Dagster Cloud users, an Enterprise contract also comes with a guaranteed SLA.
+
+### Searching for answers
+
+Search engines like Google generally do a good job at surfacing relevant Dagster docs, Github Discussions and Github Issues, so we recommend starting there.
+
+Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster, because the corpus it was trained on doesn’t include Dagster’s latest APIs.
```

Comment:
```suggestion
**Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster.** This is because the corpus it was trained on doesn’t include Dagster’s latest APIs.
```

Thoughts on bolding this to make it stand out?

Reply:
Eh I'd rather not draw too much attention to it